### PR TITLE
Fix passing license directory via ENV var to the container

### PR DESCRIPTION
**Pull request by antonovchtd**
This PR fixes two issues:
1. Verification of license activation fails due to LICENSE_DIR env variable not set.
2. Inability to set custom license directory, despite the appearance of being able to.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ ARG APP_ROOT_DIR=$DEPLOYMENT_DIR/$APP_ROOT_NAME
 ARG APP_SERVER_DIR=/opt/$APP_ROOT_NAME/AppServer
 ARG APP_NAME_MASK=wsc_app*
 ARG USER_NAME=wsc
-ARG LICENSE_DIR=/var/lib/wsc/license
+ARG SERVICE_DIR=/var/lib
```
Comment (KharchenkoMaks) [inline Dockerfile:28]: Actually, it is not the service dir

File: Dockerfile [hunk 2]
```diff
@@ -101,7 +102,7 @@ RUN tar -xvf $DEPLOYMENT_DIR/$APP_NAME_MASK -C $DEPLOYMENT_DIR/ && \
     ln -s /dev/stdout $APP_SERVER_DIR/Logs/Child-0.log && \
     ln -s /dev/stdout $APP_SERVER_DIR/Logs/Child-1.log && \
     ln -s /dev/stdout $APP_SERVER_DIR/Logs/Action.log && \
-    chown -R ${USER_ID}:${GROUP_ID} $LICENSE_DIR $DICTIONARIES_DIR $APP_SERVER_DIR
+    chown -R ${USER_ID}:${GROUP_ID} $SERVICE_DIR $DICTIONARIES_DIR $APP_SERVER_DIR
```
Comment (KharchenkoMaks) [inline Dockerfile:105]: We give permission on the whole `/var/lib`?

File: Dockerfile (hunk 1 above)
Comment (antonovchtd) [reply Dockerfile:28]: open to suggestions

File: Dockerfile (hunk 2 above)
Comment (antonovchtd) [reply Dockerfile:105]: changed to /var/lib/wsc

**Review by KharchenkoMaks — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (maxim-voronko) [inline Dockerfile:28]: Let's rename to SERVICE_FILES_DIR

**Pull request merged**